migrations/educandu-2022-12-22-01-remove-provider-from-users.js

```diff
@@ -0,0 +1,22 @@
+export default class Educandu_2022_12_22_01_remove_provider_from_users {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.collection('users').updateMany({}, { $unset: { provider: null } });
+    await this.db.collection('users').dropIndex('_idx_verificationCode_provider_');
```

Comment:
We should add an index on the single property `verificationCode` again after dropping this, because we're still filtering by it.

Reply:
I wanted to do that and found we already had a separate `_idx_verificationCode_` index.

Reply:
Oh, sorry, I didn't remember that. :+1: